web/templates/message_edit_form.hbs

```diff
@@ -1,40 +1,43 @@
 {{! Client-side Handlebars template for rendering the message edit form. }}
-
-<form id="edit_form_{{message_id}}" class="new-style">
-    <div class="edit_form_banners"></div>
-    <div class="edit-controls edit-content-container {{#if is_editable}}surround-formatting-buttons-row{{/if}}">
-        {{> copy_message_button message_id=this.message_id}}
-        <textarea class="message_edit_content" maxlength="{{ max_message_length }}">{{content}}</textarea>
-        <div class="scrolling_list preview_message_area" id="preview_message_area_{{message_id}}" style="display:none;">
-            <div class="markdown_preview_spinner"></div>
-            <div class="preview_content rendered_markdown"></div>
-        </div>
-    </div>
-    <div class="action-buttons">
-        <div class="controls edit-controls">
-            {{#if is_editable}}
-            <div class="message-edit-feature-group">
-                {{> compose_control_buttons }}
+<div class="message_edit">
+    <div class="message_edit_form">
```

Comment:
We are using the following grid template areas:
```css
grid-template-areas:
    "edited message   controls . time"
    ".      message   .        . .   "
    ".      more      .        . .   "
    ".      reactions .        . .   ";
```

The `message_edit` element is set to the `more` grid area, so it will always stay above the `message_reactions` element which occupies the `reactions` grid area.

Reply:
Just checked, and we also hide reactions before displaying the edit form, so it probably shouldn't be an issue.

Reply:
Great!